app/models/ability.rb

```diff
@@ -12,6 +12,12 @@ def initialize(user)
       can :update, Recipe do |recipe|
         recipe.user_id == user.id
       end
+      can :destroy, Food do |food|
+        food.user_id == user.id
+      end
+      can :update, Food do |food|
+        food.user_id == user.id
+      end
       can :read, :all
     end
```

Comment:
I'm not able to a new food. Kindly check your ability class and make the required changes

Reply:
Done!